src/views/DefineCompany.vue

```diff
@@ -204,7 +211,7 @@ export default class DefineCompany extends Mixins(EntityFilterMixin) {
   }
 
   private get showErrors (): boolean {
-    return Boolean(this.$route.query.showErrors)
+    return Boolean(this.$route.query.showErrors) && this.getValidateSteps
```

Comment:
Do you think we need both the route query param and the global flag? Or could we use a single mechanism for showing errors?
This is part of the "when do we show errors?" discussion we started earlier, decided to shelf, and will need to review later :)

Reply:
I removed it for now since the requirement in invisionapp is to show the error while clicking on the link from Review screen

Reply:
Ok, I was just starting the conversation about validation mechanisms.